package.json

```diff
@@ -44,6 +44,7 @@
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
+    "@types/react-modal": "^3.16.3",
```

Comment:
Would you please discard this `package.json` and `package-lock.json` file changes? Because another PR contains this changes.

Reply:
![image](https://github.com/tajnur007/d2d-marketing-frontend/assets/104530253/cb258e28-541b-48f2-ae45-5d95ec7984c3)

discard kore ei error shoho ki pr dibo?

Reply:
Yes, this error will be discarded after merging another PR.

Reply:
Ok vaiyya